### PR TITLE
Updated preprocessor

### PR DESCRIPTION
**Pull request by ScribbleTAS**
- Removed version specific accesswideners
- Updated patterns in code to include a `;`

Before I forget it, the regex to replace Patterns in the comments is:

Search:
```
// +@([^;]+)$
```
Replace
```
// @$1;
```
## TODO:
- [x] Nesting

### CODE REVIEW

File: src/main/resources/lotas.accesswidener [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+## 1.16.1
+#$$accessible field net/minecraft/server/MinecraftServer storageSource Lnet/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess;
+#$$accessible field net/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess lock Lnet/minecraft/util/DirectoryLock;
+#$$mutable field net/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess lock Lnet/minecraft/util/DirectoryLock;
+#$$accessible field net/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess levelPath Ljava/nio/file/Path;
+#$$accessible field net/minecraft/server/MinecraftServer worldData Lnet/minecraft/world/level/storage/WorldData;
+#$$mutable field net/minecraft/server/MinecraftServer worldData Lnet/minecraft/world/level/storage/WorldData;
+## def
+## end
```
Comment (PancakeTAS) [inline src/main/resources/lotas.accesswidener]: Why is there an empty "#def" block, can't it just be "#end"?

File: src/main/resources/lotas.accesswidener [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## 1.15.2
+#$$accessible field net/minecraft/world/level/chunk/storage/SectionStorage worker Lnet/minecraft/world/level/chunk/storage/IOWorker;
+#$$accessible field net/minecraft/world/level/chunk/storage/IOWorker storage Lnet/minecraft/world/level/chunk/storage/RegionFileStorage;
+#$$accessible field net/minecraft/world/entity/ai/village/poi/PoiManager loadedChunks Lit/unimi/dsi/fastutil/longs/LongSet;
+#$$accessible field net/minecraft/world/level/chunk/storage/IOWorker pendingWrites Ljava/util/Map;
+#$$accessible field net/minecraft/world/level/chunk/storage/ChunkStorage worker Lnet/minecraft/world/level/chunk/storage/IOWorker;
+#$$accessible field net/minecraft/world/level/chunk/storage/SectionStorage storage Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+## def
+## end
```
Comment (PancakeTAS) [inline src/main/resources/lotas.accesswidener]: again

File: src/main/resources/lotas.accesswidener [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+## 1.16.5
+#$$accessible field net/minecraft/world/level/storage/LevelStorageSource baseDir Ljava/nio/file/Path;
+#$$accessible field net/minecraft/world/level/storage/LevelStorageSource backupDir Ljava/nio/file/Path;
+## def
+## end
```
Comment (PancakeTAS) [inline src/main/resources/lotas.accesswidener]: again

File: src/main/resources/lotas.accesswidener [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+#$$accessible field net/minecraft/world/level/entity/EntitySectionStorage sectionIds Lit/unimi/dsi/fastutil/longs/LongSortedSet;
+#$$accessible field net/minecraft/world/level/entity/PersistentEntitySectionManager chunkVisibility Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+#$$accessible field net/minecraft/world/level/entity/PersistentEntitySectionManager chunkLoadStatuses Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+#$$accessible field net/minecraft/world/level/entity/PersistentEntitySectionManager loadingInbox Ljava/util/Queue;
+#$$accessible field net/minecraft/world/level/entity/PersistentEntitySectionManager chunksToUnload Lit/unimi/dsi/fastutil/longs/LongSet;
+#$$accessible field net/minecraft/world/level/chunk/storage/EntityStorage worker Lnet/minecraft/world/level/chunk/storage/IOWorker;
+#$$accessible field net/minecraft/world/level/entity/PersistentEntitySectionManager sectionStorage Lnet/minecraft/world/level/entity/EntitySectionStorage;
+#$$accessible field net/minecraft/world/level/entity/PersistentEntitySectionManager visibleEntityStorage Lnet/minecraft/world/level/entity/EntityLookup;
+## def
+## end
```
Comment (PancakeTAS) [inline src/main/resources/lotas.accesswidener]: again

File: src/main/resources/lotas.accesswidener [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+## 1.19.3
+#$$accessible field net/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess levelPath Ljava/nio/file/Path;
+## def
+## end
```
Comment (PancakeTAS) [inline src/main/resources/lotas.accesswidener]: again

**Review by PancakeTAS — CHANGES_REQUESTED** [review]
Fix preprocessor things I annotated

**Review by PancakeTAS — APPROVED** [review]
Thanks. Is it ready to merge now?

**Comment by ScribbleTAS** [comment]
Nope, have to apply nesting stuff to the accessors

**Pull request merged**